src/Uno.ReferenceImplComparer/Program.cs

```diff
@@ -0,0 +1,124 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Transactions;
+using Mono.Cecil;
+using Mono.Collections.Generic;
+
+namespace Uno.ReferenceImplComparer
+{
+	class Program
+	{
+		static int Main(string[] args)
+		{
+			var hasErrors = false;
+			var filePath = UnpackArchive(args[0]);
+
+			Console.WriteLine($"Validating package {args[0]}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)